[Release Guide]: Document General Availability (#734)
submariner-io/submariner-website: src/content/development/contribution-guide/_index.en.md

#### General Availability

> Once a release candidate is deemed stable and has no blocker bugs it will be released for general availability.

Reviewer: ```suggestion
Once a release candidate is deemed stable and has no blocker bugs, it will be released for general availability.
```

> Prior to releasing, the release manager will verify that there have been no changes on the stable branches since the last release

Reviewer: This release, and I think most/all releases so far, we've blocked changes that touched code or dependencies of code, but not CI (and I guess docs would be the same if we edited branched-docs). As is this is worded, it would change that practice. Is that what we want?

Author: We're talking about the stable brances, yes?
We had the same policy before more or less, it's now formal.

The website doesn't use branches so it's not affected, and CI changes backports to stable branches shouldn't be exempt automatically.

Reviewer: Okay that makes sense, thanks. I was thinking of the time we're basically "code frozen" right before cutting stable branches.

candidate.

> This ensures that the release version is as stable as possible, and avoids any bugs that might have crept in and that could pose a risk
> to the stability of the released version.

Reviewer 2: ```suggestion
This ensures that no bugs could have been introduced to possibly affect the stability of the released version.
```

The new version will be announced per the [announcement guidelines] in the release process documentation.

> The [current release board](#current-release-board) will be closed, and all open tasks on it will be moved back to the

Reviewer 2: ```suggestion
The [current release board](#current-release-board) will be closed and all remaining items will be moved back to the
```

[backlog board](#backlog-board).

> The tasks could then be considered for the next version, based on the [planning](#pre-planning) guidelines.

Reviewer 2: ```suggestion
The items can then be considered for the next version, based on the [planning](#pre-planning) guidelines.
```

> During the week when the general availability release is performed, the next version will be [planned](#pre-planning).

Reviewer: I've mentioned this elsewhere, but I think we're linking to the same things too much. Here, we link to the same thing two sentences in a row.

> Additionally, a meeting to retrospect the last release cycle will be held.

Reviewer 2: ```suggestion
Additionally, a retrospective meeting for the last release cycle will be held.
```

Reviewer 3: I think L121 and L122 are not relevant here. All agile / scrum meetings are detailed in `Sprints` section of this guide, while this section belongs to the GA process

Author: > I think L121 and L122 are not relevant here. All agile / scrum meetings are detailed in `Sprints` section of this guide, while this section belongs to the GA process

This is a general retrospective for the entire cycle, or do you think we dont need it?
